packages/components/psammead-grid/src/index.jsx

```diff
@@ -110,31 +112,142 @@ const gridMediaQueries = ({
   );
 };
 
+const startOffsetPercentage = (columnsGroup, gridStartOffsetGroup) =>
+  `${(100 / columnsGroup) * (gridStartOffsetGroup - 1)}%`;
+
+const negativeOffset = (
+  columnsGroup,
+  parentColumnsGroup,
+  gridStartOffset,
+  gridStartOffsetGroup,
+) => {
+  const isValidOffset =
+    gridStartOffset &&
+    gridStartOffsetGroup &&
+    gridStartOffsetGroup < parentColumnsGroup &&
+    columnsGroup === parentColumnsGroup;
+
+  return isValidOffset
+    ? ` - ${startOffsetPercentage(parentColumnsGroup, gridStartOffsetGroup)}`
+    : ``;
+};
+
 /*
  * 1 We vertically align to the top so that sibling
  *   grid items that are placed side-by-side within a row
  *   have their text and images aligned
  */
+const childrenFallback = (
+  item,
+  dir,
+  columnsGroup,
+  parentColumnsGroup,
+  parentEnableGelGutters,
+  gutterSize,
+  gridStartOffset,
+  gridStartOffsetGroup,
+) => ` 
+  ${
+    item && parentEnableGelGutters
+      ? ` 
+        margin: 0 ${parseFloat(gutterSize) / 2}rem;
+        width: calc(${(100 * columnsGroup) / parentColumnsGroup}%
+          - ${gutterSize}${negativeOffset(
+          columnsGroup,
+          parentColumnsGroup,
+          gridStartOffset,
+          gridStartOffsetGroup,
+        )});
+        `
+      : `
+        width: calc(${(100 * columnsGroup) /
+          parentColumnsGroup}%${negativeOffset(
+          columnsGroup,
+          parentColumnsGroup,
+          gridStartOffset,
+          gridStartOffsetGroup,
+        )});
+        `
```

Comment:
Can we refactor this and  have a variable set to the results of 👇 
```js
negativeOffset(
  columnsGroup,
  parentColumnsGroup,
  gridStartOffset,
  gridStartOffsetGroup,
)})
```

Reply:
Done!